### DISCUSSION TITLE
Cloudflare worker deploy support

### DISCUSSION THREAD

**Discussion by janat08** [oven-sh/bun#971] [Q&A]
Can this be deployed to cloudflare workers? Workers pretty much need a module that has a fetch export. I can't find how to create a bundle to try and deploy with wrangler. Also is the bun bundle smaller than 1mb, because thats the limit for cloudflare worker bundle.

**Comment by enqqi** [comment]
@janat08 there was an issue opened by Jarred for this

https://github.com/oven-sh/bun/issues/121
